draft-ietf-anima-brski-prm.md

```diff
@@ -1103,123 +1163,126 @@ pledgeenrollrequesttrigger = {
-The registrar MAY consider to ignore any but the newest PER from the same pledge in the case the registrar has at any point in time more than one pending PER from the pledge.
+While BRSKI-PRM targets the initial enrollment, re-enrollment can be supported in a similar way.
+In this case, the pledge MAY use its current LDevID credential instead of its IDevID credential to sign the PER artifact.
+The issuing CA can associate the re-enrollment request with the pledge based on the previously issued and still valid LDevID certificate.
+Note that a pledge that does not have synchronized time needs to advance the last known current date and time based on its local clock over a longer period, which also requires persisting the local clock advancements across reboots.
 
-As the Registrar-Agent is intended to facilitate communication between the pledge and the domain registrar, a collection of requests from more than one pledge is possible.
-This allows bulk bootstrapping of several pledges using the same connection between the Registrar-Agent and the domain registrar.
 
 
+## Supply PVR to Registrar (including MASA interaction) {#pvr}
 
-## Supply PVR to Registrar (including backend interaction) {#pvr}
+Once the Registrar-Agent has acquired one or more PVR and PER object pairs, it starts the interaction with the domain registrar.
+Collecting multiple pairs allows bulk bootstrapping of several pledges using the same session with the registrar.
 
-Similar to BRSKI "requestvoucher" endpoint in {{Section 5.2 of !RFC8995}}.
+The Registrar-Agent MUST establish a TLS session to the registrar with mutual authentication.
+In contrast to BRSKI {{RFC8995}}, the TLS client authentication uses the Registrar-Agent LDevID certificate instead of pledge IDevID certificate.
+Consequently, the domain registrar can distinguish BRSKI (pledge-initiator-mode) from BRSKI-PRM (pledge-responder-mode).
 
-The Registrar-Agent has acquired one or more PVR and PER object pairs
+The registrar SHOULD verify the TLS client authentication of the Registrar-Agent.
+Note that authentication and authorization is verified during the TLS session based on the signatures inside the PVR artifact.
 
-The Registrar-Agent establishes a TLS connection to the registrar.
 As already stated in {{!RFC8995}}, the use of TLS 1.3 (or newer) is encouraged.
 TLS 1.2 or newer is REQUIRED on the Registrar-Agent side.
 TLS 1.3 (or newer) SHOULD be available on the registrar, but TLS 1.2 MAY be used.
 TLS 1.3 (or newer) SHOULD be available on the MASA, but TLS 1.2 MAY be used.
 
-In contrast to BRSKI {{RFC8995}} TLS client authentication to the registrar is achieved by using Registrar-Agent EE credentials instead of pledge IDevID credentials.
-Consequently BRSKI (pledge-initiator-mode) is distinguishable from BRSKI-PRM (pledge-responder-mode) by the registrar.
```

Comment:
Moved to the corresponding place in the exchange flow.

Reply:
okay